lib/parallel_tests/cli.rb

```diff
@@ -13,6 +13,8 @@ def run(argv)
       num_processes = ParallelTests.determine_number_of_processes(options[:count])
       num_processes = num_processes * (options[:multiply] || 1)
 
+      options[:use_test_env_number_for_first_process] ||= ParallelTests.always_use_test_env_number_for_first_process?
```

Comment:
I'd prefer to keep it local unless it's useful for the public or other
modules

On Tue, Jan 19, 2016 at 11:00 AM, Rhett Sutphin <notifications@github.com>
wrote:

> In lib/parallel_tests/cli.rb
> <https://github.com/grosser/parallel_tests/pull/477#discussion_r50157098>:
>
> > @@ -13,6 +13,8 @@ def run(argv)
> >        num_processes = ParallelTests.determine_number_of_processes(options[:count])
> >        num_processes = num_processes * (options[:multiply] || 1)
> >
> > +      options[:use_test_env_number_for_first_process] ||= ParallelTests.always_use_test_env_number_for_first_process?
>
> It's ||= to allow for the option to be set on the command line as well.
>
> I like having the check code in ParallelTests because it's a similar
> construction to ParallelTests.determine_number_of_processes. I can change
> it if you prefer, though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/parallel_tests/pull/477/files#r50157098>.
>